report/pdf_report.go

```diff
@@ -119,6 +119,32 @@ func RunPdfReport(outputPath string) error {
 		block.Draw(p)
 	})
 
+	// Draw side note on each content page
+	c.PageFinalize(func(args creator.PageFinalizeFunctionArgs) error {
+		if args.PageNum < 3 {
+			return nil
+		}
+
+		p := c.NewStyledParagraph()
```

Comment:
can you create a separate example for this perhaps?  it does not make the report look particularly nice.